test/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is there a reason why we have a separate `eslint` configuration for tests?

Reply:
separation of concern.

Reply:
Does the configuration differ? 

1. They shouldn't differ.
2. If they don't differ, then there might not be a need to separate concerns.

Reply:
You have a point there.
`eslintrc.json` has rules that concern `chai` and `cypress` although I don't know how these rules would interact with other unrelated components.

Do have a look at them. 
`eslint` config for test dir:
<img width="478" alt="Screenshot 2019-12-03 at 5 54 39 PM" src="https://user-images.githubusercontent.com/29008971/70071969-69d61f80-15f6-11ea-8768-a217fcef765e.png">

global config: 
<img width="553" alt="Screenshot 2019-12-03 at 5 54 45 PM" src="https://user-images.githubusercontent.com/29008971/70071927-575be600-15f6-11ea-9d49-8c83e49c2eb6.png">

Reply:
We can merge this file with the `.eslintrc` file in the root of this project since there are no conflicting rules. It's redundant having two `eslint` files for now.